README.md

```diff
@@ -11,316 +11,264 @@
-   the check, as explained below).
-
-### Condition
-
-A condition is a function that can be added to a Check, which returns a
-pass ✓, fail ✖ or warning ! result, intended for validating the Check's
-return value. An example for adding a condition would be:
-
-```python
-from deepchecks.tabular.checks import BoostingOverfit
-BoostingOverfit().add_condition_test_score_percent_decline_not_greater_than(threshold=0.05)
-```
-
-which will return a check failure when running it if there is a difference of
-more than 5% between the best score achieved on the test set during the boosting
-iterations and the score achieved in the last iteration (the model's "original" score
-on the test set).
-
-### Suite
-
-An ordered collection of checks, that can have conditions added to them.
-The Suite enables displaying a concluding report for all of the Checks
-that ran.
-
-See the list of [predefined existing suites](deepchecks/tabular/suites)
-for tabular data to learn more about the suites you can work with
-directly and also to see a code example demonstrating how to build your
-own custom suite.
-
-The existing suites include default conditions added for most of the checks.
-You can edit the preconfigured suites or build a suite of your own with a collection
-of checks and optional conditions.
+<details open>
+   <summary>
+      <h4>
+         Deepchecks CI & Testing Management Quickstart
+      </h4>
+   </summary>
 
+Deepchecks managed CI & Testing management is currently in closed preview.
+[Book a demo](https://deepchecks.com/book-demo/) for more information about the offering.
 
 <p align="center">
-   <img src="/docs/source/_static/images/general/diagram.svg">
+   <img src="docs/source/_static/images/general/deepchecks-ci-checks.png" width="800">
 </p>
 
+For building and maintaining your own CI process while utilizing Deepchecks Testing for it,
+check out our [docs for Using Deepchecks in CI/CD](https://docs.deepchecks.com/stable/general/usage/ci_cd.html).
 
-## 🤔 What Do You Need in Order to Start Validating?
-
-### Environment
-
-- The deepchecks package installed
-- JupyterLab or Jupyter Notebook or any Python IDE
-
-
-### Data / Model 
-
-Depending on your phase and what you wish to validate, you'll need a
-subset of the following:
-
--  Raw data (before pre-processing such as OHE, string processing,
-   etc.), with optional labels
--  The model's training data with labels
--  Test data (which the model isn't exposed to) with labels
--  A [supported
-    model](https://docs.deepchecks.com/stable/user-guide/supported_models.html?utm_source=github.com&utm_medium=referral&utm_campaign=readme&utm_content=running_a_suite) (e.g. scikit-learn models, XGBoost, any model implementing the ``predict`` method in the required format)
+</details>
 
+## 📜 Open Source vs Paid 
 
-### Supported Data Types
+Deepchecks' projects (``deepchecks/deepchecks`` & ``deepchecks/monitoring``) are open source and are released under [AGPL 3.0](./LICENSE).
```

Comment:
Make it clearer that THIS current repo (``deepchecks/deepchecks``) is 100% AGPL free forever.

Reply:
On purpose this readme doesn't distinct between the repos so it may be confusing to relate to that.
Did make the phrasing in the next sentence a bit direct.

Also wouldn't want to commit on that, in case we add in the future regulation-related checks that need a license (and are likely to be in deepchecks/deepchecks)